{"name": "登录反向用例测试", "status": "broken", "statusDetails": {"message": "KeyError: 'error_mag'", "trace": "self = <cases.test_login.TestLogin object at 0x000001F62B9E95F8>\nhandler = (<appium.webdriver.webdriver.WebDriver (session=\"db338935-8efb-4890-b2be-199f4d9dde3a\")>, <pages.page_action.PageAction object at 0x000001F62B9E9D30>)\ndata = {'error_msg': '用户名不能为空！', 'name': '登录-反向用例-用户名为空', 'password': '654321', 'username': ''}\n\n    @allure.story('钓鱼人APP自动化测试-登录模块-反向用例测试')\n    @allure.title('登录反向用例测试')\n    @pytest.mark.parametrize('data',failed_data)\n    def test_login_failed(self, handler, data):\n        logger.info('开始执行反向测试用例：{}'.format(data['name']))\n        handler[1].login(data['username'], data['password'])\n        logger.info('登录操作完成')\n>       assert handler[1].is_toast_show(data['error_mag'])\nE       KeyError: 'error_mag'\n\ncases\\test_login.py:37: KeyError"}, "attachments": [{"name": "log", "source": "87f05560-a090-4105-9fd9-429eee615271-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'name': '登录-反向用例-用户名为空', 'username': '', 'password': '654321', 'error_msg': '用户名不能为空！'}"}], "start": 1615258232227, "stop": 1615258233928, "uuid": "530027cb-f037-4af3-adaa-622e28ada199", "historyId": "d93c8b521d1ee7fc4eb27338522fd359", "testCaseId": "130e6dce60bc4ef9d882c5f0465dc1d6", "fullName": "cases.test_login.TestLogin#test_login_failed", "labels": [{"name": "story", "value": "钓鱼人APP自动化测试-登录模块-反向用例测试"}, {"name": "feature", "value": "钓鱼人APP自动化测试-登录模块"}, {"name": "epic", "value": "钓鱼人APP自动化测试"}, {"name": "tag", "value": "@pytest.mark.usefixtures('prepare_to_login')"}, {"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-LSM6I36P"}, {"name": "thread", "value": "12032-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.test_login"}]}